packages/frontend/src/utils/fetch.ts

```diff
@@ -19,10 +22,11 @@ export async function fetchApi<TPath extends keyof AuthRoutes, TMethod extends k
 	const res = await fetch(`${process.env.NEXT_PUBLIC_API_URL!}${path}`, {
 		method: method as string,
 		body: body as BodyInit,
+		credentials: 'include',
 	});
 
 	if (res.status >= 200 && res.status < 300) {
-		return res.json() as Promise<InferAuthRouteResult<TPath, TMethod>>;
+		return (await res.json()) as InferAuthRouteResult<TPath, TMethod>;
```

Comment:
Prefer how this was before. `return await` is redundant unless in a `try` block.